luigi/target.py

```diff
@@ -23,8 +23,15 @@
 import io
 import os
 import random
+import shutil
 import tempfile
 import logging
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
Oh I see. Hmm, I wouldn't dare touch it, kinda sad that it's not commented. Oh well.